### DISCUSSION TITLE
Sound ques on incoming turn

### DISCUSSION THREAD

**Discussion by exus1pl** [osmandapp/OsmAnd#16925] [Ideas]
Hi,
Would it be possible to instead of voice turn messages to play a sound ques when coming in to a turn? Most of the bicycle navigation units on the marked have such feature and it is quit nice, especially when riding in the wild.

**Comment by mikehgentry** [comment]
Hi,

There are a few beep voice packs available, designed especially for cycling. 

Navigation -> Settings -> Voice prompts (settings) -> Language -> Recorded

Minimal basically just lets you know if you go off course, get back on course, or arrive at your destination. Simple beeps to let you know when to turn as well. Complex gives you turn warnings ahead of time (short beep means prepare, long beep means do something now), and tells you which direction to go - low means left, high means right. A series of middle pitched beeps tells you which exit to take on a roundabout or something. And all three pitches means do a u-turn.

I've found they work well with announcement time set to early in the Voice prompts settings screen.

Feedback is welcome if you can suggest any improvements. Most of the discussion so far has been in this thread and a few linked to it #10501

**Comment by exus1pl** [comment]
Wow, thanks, I didn't see them there, that is perfect! Looks like when I was searching I didn't thing on searching for beeps :)
